### PR TITLE
feat(lspsaga): support v0.3

### PR DESCRIPTION
**Pull request by Jint-lzxy**

### CODE REVIEW

**Review by ayamir — APPROVED** [review]
LGTM 👍

File: lua/modules/configs/completion/lspsaga.lua [hunk 1]
```diff
@@ -26,35 +26,42 @@ return function()
...
+				vsplit = "<C-c>v",
+				split = "<C-c>s",
+				tabe = "<C-c>t",
+				close = "<C-c>q",
+				quit = "q",
+			},
 		},
 		code_action = {
 			num_shortcut = true,
+			show_server_name = false,
```
Comment (CharlesChiuGit) [inline lua/modules/configs/completion/lspsaga.lua:63]: I think it would be better to set it to `true`?

**Review by CharlesChiuGit — CHANGES_REQUESTED** [review]

File: lua/modules/configs/completion/lspsaga.lua (hunk 1 above)
Comment (Jint-lzxy) [reply lua/modules/configs/completion/lspsaga.lua]: Agree lol

https://github.com/ayamir/nvimdots/pull/857/commits/4b8b336c0c72e6294f066c5d0b0ef9c9f8e6dbde

**Review by CharlesChiuGit — APPROVED** [review]
LGTM

**Pull request merged**